builtins/amp-ad.md

```diff
@@ -1,6 +1,7 @@
 # <a name="amp-ad"></a> `amp-ad`
 
-NOTE: The specification of `amp-ad` is likely to significantly evolve over time. The current approach is designed to bootstrap the format to be able to show ads.
+NOTE: To use `<amp-ad>`, the script to the `amp-ad` library is needed. It is recommended to add the script manually but currently it will be automatically fetched when `amp-ad` is used.
```

Comment:
My sense is that we can just delete this. We basically want to just document what should be done now. Maybe put a note at the bottom for existing users.